charts/nginx-website/tests/defaults_test.yaml

```
@@ -33,3 +33,31 @@ tests:
           value: {}
       - notExists:
           path: spec.template.spec.containers[0].resources
+  - it: should create a default nginx configuration
+    template: nginx-configmap.yaml
+    asserts:
+      - hasDocuments:
+          count: 1
+      - isKind:
+          of: ConfigMap
+      - equal:
+          path: metadata.name
+          value: RELEASE-NAME-nginx-website
+      - matchRegex:
+          path: data["default.conf"]
+          pattern: 'listen       80;'
+      - matchRegex:
+          path: data["default.conf"]
+          pattern: 'server_name  localhost;'
+      - matchRegex:
+          path: data["default.conf"]
+          pattern: 'root   /usr/share/nginx/html;'
+      - matchRegex:
+          path: data["default.conf"]
+          pattern: 'index  index.html index.htm;'
+      - matchRegex:
+          path: data["default.conf"]
+          pattern: 'autoindex on;'
+      - notMatchRegex:
+          path: data["default.conf"]
+          pattern: 'try_files $uri /index.html;'
```

```suggestion
          pattern: 'try_files \$uri /index.html;'
```
Forgot to fix this (non-matching) regexp.